setup.py

```diff
@@ -29,6 +29,7 @@
     ],
     "docs": [
         "sphinx==4.3.1",
+        "sphinx_autodoctypehints==1.19.2",
```

Comment:
No napoleon here? It'll get pulled in automatically as a depdency, I guess?

Reply:
Apparently is not installed, but just [pulled in after installing sphinx](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html)

Reply:
I misread the diff, thought you were adding it.